microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -34,11 +34,19 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `productID` | Required.  The product ID of the product to which the package must be uploaded. |
 
-###### Request headers
+###### Request headers (v1)
 
-*  Required.  `Authorization: Bearer <auth token>`
+* Required.  `Authorization: Bearer <auth token>`
```

Comment:
```suggestion
The following headers are required:

* `Authorization: Bearer <auth token>`
```
Every header that's listed in the article is Required; the word Optional doesn't occur in the article.  Therefore I am considering this improved format, to present each header isolated as a list item.

Reply:
I implemented this formatting, to clarify syntax of headers; also tightened list spacing since items are short & don't wrap.